lib/ecs_helper/command/export_images.rb

```diff
@@ -15,7 +15,9 @@ def required
   end
 
   def run
-    puts export_images
+    variables = export_images
+    puts variables
```

Comment:
remove debug

Reply:
It is not a debug, it puts export to STDOUT